src/fronts/film-today/FilmToday.tsx

```diff
@@ -2,15 +2,17 @@ import React from "react";
 import { Collection as ICollection } from "../../api";
 import { Collections } from "../../components/Collections";
 import { Collections as CollectionsVariantB } from "./variantB/Collections";
-import { Collections as CollectionsVariantC } from "./variantC/Collections";
+import { Collections as CollectionsVariantC } from "./variantZ/Collections";
 
 export const FilmToday: React.FC<{
     frontId: string;
     collections: ICollection[];
     salt: string;
     variant?: string;
 }> = ({ frontId, collections, salt, variant }) => {
-    if (variant === "c") {
+    if (variant === "z") {
```

Comment:
What are your thoughts on just removing these now dead variants?

Reply:
We want to keep them for now as otherwise we can't see the collections or any of the components they use. The plan is to move all collections to Storybook so we can see them there and delete the variants from the app.

Reply:
"Not seeing the collections" is not a huge problem in itself, but we need to do a bit of testing on them every now and then to make sure they weren't broken but changes made to the components - that way we're confident all containers we've build are still functional (whether they're currently in use or not).